CHANGELOG.md

```diff
@@ -91,7 +91,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - [#228 - show selectbox options](https://github.com/shopsys/shopsys/pull/228)
     - the use of jQuery plugin for selectboxes was modified on Shopsys Framework side so options will now be seen 
 - [#243 - Admin: changed domain icon in e-shop domain administration can be saved](https://github.com/shopsys/shopsys/pull/243)
-    - changed domain icon in e-shop domain administration can be saved 
+    - changed domain icon in e-shop domain administration can be saved
+- copying of localized entities
+    - detection of new locale is now done before multidomain data are created 
```

Comment:
If someone asks me, why this change was necessary, I can not answer. Can you explain it to me? Why was it necessary? Why is the previous state no longer correct?

Reply:
it is necessary when new locale is added but it is not necessary in this PR so I am removing this change :+1:

Reply:
it is necessary when new domain with new locale is added :slightly_smiling_face: